src/server/accounting.c

```diff
@@ -2259,3 +2259,86 @@ void log_alter_records_for_attrs(job *pjob, svrattrl *plist) {
 	if (entire_record[0] != '\0')
 		account_record(PBS_ACCT_ALTER, pjob, entire_record);
 }
+
+
+/**
+ * @brief
+ * Common function to log a suspend/resume record 
+ * for suspend/resume job events respectively.
+ * 
+ * @param[in] pjob - job to log records for.
+ * @param[in] user - User who requested the signal suspend/resume 
+ * @param[in] host - Hostname where the signal was triggered
+ * @param[in] signal_type - flag indicating the type of event suspend/resume
+ * 
+ * @returns void 
+ */
+void
+log_suspend_resume_record(job *pjob, char *user, char *host, char *signal_type)
```

Comment:
instead of a char* signal_type, how about a "int acct_type", which can be passed PBS_ACCT_RESUME or  PBS_ACCT_SUSPEND? int comparisons are faster than string comparisons